.gitlab-ci.yml

```diff
@@ -114,6 +114,7 @@ before_script:
   script:
     - cd test-suite
     - make clean
+    - rm -rf coq-makefile/findlib-package # randomly fails
```

Comment:
What kind of ridiculous comment is "randomly fails"? 

All it communicates to me is that you are ignorant about what your system is doing.

Reply:
You are clearly not very good at communication, so it seems.